engine/src/test/java/io/camunda/zeebe/engine/processing/job/FailJobTest.java

```diff
@@ -315,4 +317,23 @@ public void shouldFailWithVariables() {
         .hasName("foo")
         .hasValue("\"bar\"");
   }
+
+  @Test
+  public void shouldTruncateErrorMessage() {
+    // given
+    final Record<JobRecordValue> job = ENGINE.createJob(jobType, PROCESS_ID);
+    final String exceedingErrorMessage = "*".repeat(501);
```

Comment:
🙃 A bit nitpicky, but I would refer having a constant here indicating the `MAX_MESSAGE_SIZE` and changing this to `"*".repeat(MAX_MESSAGE_SIZE + 1);`. This makes it clear in the tests that 500 is the max message size and we are exceeding it here.

Reply:
You can then also use this in the assertions